src/classes/Room.js

```diff
@@ -1,3 +1,9 @@
 export default class Room {
+  constructor(roomDataObject) {
+      this.roomNumber = rooms.roomNumber;
+    this.roomType = rooms.roomType;
+
```

Comment:
I usually put my export default class on the bottom of the files, but as long as it's being imported to your scripts file.